### DISCUSSION TITLE
Adding support for Code Hike

### DISCUSSION THREAD

**Discussion by hrishiksh** [withastro/roadmap#470] [Proposal]
### Body

[Code Hike](https://codehike.org/) is a nice tool to add code snippets in a website. It also offer many useful feature that make your simple blog into a storytelling page.

This only support react and its meta-frameworks. Can you please make an Astro extension for it. I really like astro and want to see this feature by default for astro.

**Comment by bluwy** [comment]
Shouldn't Code Hike support Astro instead? It mentioned that it's a remark plugin for MDX, which you can configured through [extendMarkdownConfig](https://docs.astro.build/en/guides/integrations-guide/mdx/#extendmarkdownconfig). If it injects React specific code, I don't think there's anything we can do here until it can be agnostic.
